### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by UlisesGascon**
# Description

### New NPM tasks (264ec6f):
- Audit dependencies
- Audit security in dependencies
- Upgrade dependencies

### New dependencies added (264ec6f):
- `snyk`
- `npm-check-updates`

### Update Git Hooks (2784ee7)
- Added dependencies validation to pre-push hook

### Dependencies upgraded (696400b)

```
 chalk                                      2.4.2  →    3.0.0 
 enquirer                                  ^2.3.2  →   ^2.3.4 
 lighthouse                                 5.2.0  →    5.6.0 
 ora                                        4.0.2  →    4.0.3 
 yargs                                     14.2.0  →   15.1.0 
 @commitlint/cli                           ^7.2.1  →   ^8.3.5 
 @commitlint/config-conventional           ^7.1.2  →   ^8.3.4 
 @semantic-release/changelog               ^3.0.6  →   ^5.0.0 
 @semantic-release/git              ^7.1.0-beta.9  →   ^9.0.0 
 babel-eslint                             ^10.0.1  →  ^10.0.3 
 babel-preset-env                          ^1.6.1  →   ^1.7.0 
 cz-conventional-changelog                 ^1.2.0  →   ^3.1.0 
 eslint                                    ^6.0.1  →   ^6.8.0 
 eslint-config-standard                   ^13.0.1  →  ^14.1.0 
 eslint-plugin-import                     ^2.18.0  →  ^2.20.0 
 eslint-plugin-jest                       ^22.7.2  →  ^23.6.0 
 eslint-plugin-node                        ^9.1.0  →  ^11.0.0 
 eslint-plugin-standard                    ^4.0.0  →   ^4.0.1 
 husky                                     ^3.0.0  →   ^4.2.1 
 jest                                     ^24.8.0  →  ^25.1.0 
 lint-staged                               ^9.2.0  →  ^10.0.7 
 lockfile-lint                             ^2.0.1  →   ^3.0.9 
 prettier                                 ^1.18.2  →  ^1.19.1 
 semantic-release                 ^16.0.0-beta.42  →  ^17.0.1 
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

I didn't open an issue yet, sorry 😓. I was on my way to solve #49

## Motivation and Context

- Let's simplify the dependencies outdated with some NPM Scripts and git hooks :muscle:
- Let's update old dependencies and remove possible vulnerabilities

## How Has This Been Tested?

No extra tests were required ;-)

## Checklist:

- [x] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I added a picture of a cute animal cause it's fun

**Cute animal pic**
cute animal

### CODE REVIEW

**Comment by lirantal** [comment]
Thanks for this!

@UlisesGascon I'm good with all the package updates except for the changes that introduce npm audit and snyk as a run script (and snyk as dep) so let's remove these and the pre-push run-scripts. The reason for that is that I anyway monitor the project with Snyk for any vulnerabilities introduced in a PR:
image


As well as I have Snyk set to open Pull Requests to update dependencies (cc https://dev.to/snyk/making-automatic-yet-informed-decisions-when-upgrading-dependencies-5ij on how this works)

**Comment by UlisesGascon** [comment]
Makes sense! I didn't check the CI before I submitted the changes. I will fix that asap :-)

**Comment by UlisesGascon** [comment]
@lirantal (7158f32) Let me know if I need to made more changes :-)

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "coverage:view": "open-cli coverage/lcov-report/index.html",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "deps:upgrade": "ncu -u && npm install"
```
Comment (lirantal) [inline package.json]:
```suggestion
```

Let's remove this too ;-)

File: package.json [hunk 2]
```diff
@@ -42,39 +43,40 @@
...
-    "husky": "^3.0.0",
-    "jest": "^24.8.0",
-    "lint-staged": "^9.2.0",
-    "lockfile-lint": "^2.0.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "husky": "^4.2.1",
+    "jest": "^25.1.0",
+    "lint-staged": "^10.0.7",
+    "lockfile-lint": "^3.0.9",
+    "npm-check-updates": "^4.0.1",
```
Comment (lirantal) [inline package.json]:
```suggestion
    "npm-check-updates": "^4.0.1",
```

this too

Comment (UlisesGascon) [reply package.json]: Fixed at a7b5775

File: package.json (hunk 1 above)
Comment (UlisesGascon) [reply package.json]: Fixed at a7b5775

**Comment by UlisesGascon** [comment]
@lirantal sorry for the extra commit (7e00038) but Code and Git Kraken are not sync very well today 😅

**Comment by lirantal** [comment]
No worries at all, I can also rebase it from the PR so it's all good :)

**Pull request merged**